app/resources/v1/ont/flowcell_resource.rb

```diff
@@ -2,10 +2,26 @@
 
 module V1
   module Ont
-    # FlowcellResource
+    # @todo This documentation does not yet include a detailed description of what this resource represents.
+    # @todo This documentation does not yet include detailed descriptions for relationships, attributes and filters.
+    # @todo This documentation does not yet include any example usage of the API via cURL or similar.
+    #
+    # @note Access this resource via the `/api/v1/ont/flowcell/` endpoint.
```

Comment:
Might be worth checking all the @note endpoints as I don't _think_ they include `api`. e.g. I think `https://<traction>/v1/ont/flowcells` is valid but not `https://<traction>/api/v1/ont/flowcells`
 Also if an example endpoint is added for the singular flowcell maybe add an `/<id>`, or maybe us the plural to access all the resources (although might be a not performant!)

Reply:
I think you're right! I'll remove the `/api` bit. I probably hijacked all the `api` things from Sequencescape.